moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -66,21 +66,21 @@ if(BUILD_TESTING)
   ament_add_gtest(test_bullet_collision_detection test/test_bullet_collision_detection_pr2.cpp)
   target_link_libraries(test_bullet_collision_detection moveit_test_utils ${MOVEIT_LIB_NAME})
   # TODO: remove if transition to gtest's new API TYPED_TEST_SUITE_P is finished
-  if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  if(CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)")
```

Comment:
A single `add_compile_options` call would let you avoid repeating this same snippet in this directory.
```cmake
if(CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)")
    add_compile_options(-Wno-deprecated-declarations)
endif()
```